compiler/rustc_target/src/spec/targets/aarch64_unknown_fuchsia.rs

```diff
@@ -13,6 +15,11 @@ pub(crate) fn target() -> Target {
         data_layout: "e-m:e-p270:32:32-p271:32:32-p272:64:64-i8:8:32-i16:16:32-i64:64-i128:128-n32:64-S128-Fn32".into(),
         arch: "aarch64".into(),
         options: TargetOptions {
+            pre_link_args: TargetOptions::link_args(LinkerFlavor::Gnu(Cc::No, Lld::No), &[
+                "--execute-only",
+                // Enable the Cortex-A53 errata 843419 mitigation by default
+                "--fix-cortex-a53-843419",
+            ]),
             features: "+v8a".into(),
```

Comment:
For aarch64 we should set `+crc,+aes,+sha2`.

Reply:
Fixed in the latest version.